iron-overlay-behavior.html

```diff
@@ -80,8 +80,8 @@
        * Set to true to display a backdrop behind the overlay.
        */
       withBackdrop: {
-        type: Boolean,
-        value: false
+        observer: '_withBackdropChanged',
+        type: Boolean
```

Comment:
we never really check if is `false`, so it should be a bit more performant to have one setter less.